scripts/trading_strategy_helpers.js

```diff
@@ -272,29 +276,8 @@ const buildOrderTransactionData = async function(
     const lowerLimit = lowestLimit + safeIndex * stepSize
     const upperLimit = lowestLimit + (safeIndex + 1) * stepSize
 
-    // Sell targetToken for stableToken at targetTokenPrice = upperLimit
-    // Sell 1 ETH at for 102 DAI (unlimited)
-    // Sell x ETH for max256 DAI
-    // x = max256 / 102
-    const sellPrice = formatAmount(upperLimit, targetToken)
-    // sellPrice = 102000000000000000000
-    const upperSellAmount = max128
-      .div(sellPrice)
-      .mul(toETH(1))
-      .toString()
-    const upperBuyAmount = max128.toString()
-
-    // Buy ETH at 101 for DAI (unlimited)
-    // Sell stableToken for targetToken in at targetTokenPrice = lowerLimit
-    // Sell 101 DAI for 1 ETH
-    // Sell max256 DAI for x ETH
-    // x = max256 / 101
-    const buyPrice = formatAmount(lowerLimit, targetToken)
-    const lowerSellAmount = max128.toString()
-    const lowerBuyAmount = max128
-      .div(buyPrice)
-      .mul(toETH(1))
-      .toString()
+    const [upperSellAmount, upperBuyAmount] = calculateBuyAndSellAmountsFromPrice(upperLimit, targetToken)
+    const [lowerBuyAmount, lowerSellAmount] = calculateBuyAndSellAmountsFromPrice(lowerLimit, targetToken)
```

Comment:
I see that the output of calculateBuyAndSellAmountsFromPrice is [sellAmount, buyAmount], but in the second line sell and buy are inverted. Is this expected?

Reply:
I added a comment for explaintion